File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        l = 0
        r = len(numbers)
        while l < r:
            if (numbers[l] + numbers[r-1]) == target:
                return [l+1, r]
            elif (numbers[l] + numbers[r-1]) < target:
                l += 1
            else:
                r -= 1

File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int l = 0;
        int r = numbers.length;
        while(l < r){
        if(numbers[l] + numbers[r-1] == target){
            return new int [] {l + 1, r};
        }else if (numbers[l] + numbers[r-1] < target){
            l++;
        }else{
            r--;
        }
    }
    return new int [] {};
    }
}